### PR TITLE
tick text description typo

### PR DESCRIPTION
**Pull request by cldougl**

### CODE REVIEW

File: src/plots/cartesian/layout_attributes.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ module.exports = {
         description: [
             'Sets the text displayed at the ticks position via `tickvals`.',
             'Only has an effect if `tickmode` is set to *array*.',
```
Comment (cldougl) [inline src/plots/cartesian/layout_attributes.js]: actually while editing this, could I rm 'Only has an effect if `tickmode` is set to *array*.' cause that's actually not true anymore right @etpinard // @mdtusz

Comment (etpinard) [inline src/plots/cartesian/layout_attributes.js]: It depends what we mean by _set_.

Does _set_ mean:

1. that `tickmode` **has to be set** by the user to `'array'`, or
1. that `tickmode` is **either** set by the user to `'array'` or inferred to be `'array'` by [`tickvals`](https://github.com/plotly/plotly.js/blob/071474781273374eef4beca0298aed60929a2fe6/src/plots/cartesian/tick_value_defaults.js#L23).

I think item 1. is probably what most people would understand by _set_. But how should we describe situation similar to item 2. ?

Comment (cldougl) [inline src/plots/cartesian/layout_attributes.js]: ahh yes you're totally right, I wasn't thinking about opt 2. 
We should probably leave as is for now then. (but just change the typo in this commit)

**Comment by etpinard** [comment]
Great. :dancer:

**Pull request merged**